ldm/invoke/concepts_lib.py

```diff
@@ -27,10 +27,12 @@ def __init__(self, root=None):
         self.match_trigger = re.compile('(<[\w\- >]+>)') # trigger is slightly less restrictive than HF concept name
         self.match_concept = re.compile('<([\w\-]+)>') # HF concept name can only contain A-Za-z0-9_-
 
-    def list_concepts(self)->list:
+    def list_concepts(self, minimum_likes: int=0)->list:
```

Comment:
That's along the right lines. To really do it right, we need to do some scaling to avoid a bias against more recent models that haven't had enough time to accumulate likes. Right now I'm happy to see how often we get complaints that we missed a good model that should have been on the list.

In 3.0, the WebUI dropdowns will disappear and we'll be able to autocomplete. The like threshold can go away at that point.